monitoring/.dockerignore

```diff
@@ -0,0 +1,20 @@
+# Hidden files and directories.
+.*
+
+# Example
+example/
+
+# Configuration
+config/
+
+# Documentation
+docs/
+out/
+README.adoc
+
+# Node
+external/
+node_modules/
+
+# Make
+Makefile
```

Comment:
Should we cleanup this file a bit? For example:
- there is no `Makefile` for this module,
- we have neither `example` nor `config`.

Reply:
See https://github.com/keep-network/tbtc-v2/pull/535/commits/48c3768fd09a58d896b4a50f63e37b2fc34b0050